doc/source/cluster/index.rst

```diff
@@ -0,0 +1,107 @@
+.. _cluster-index:
+
+Multi-node Ray Overview
+=======================
+
+One of Ray's strengths is the ability to leverage multiple machines in the same program. You can do this by creating and connecting to a Ray cluster.
+
+Key Concepts
+------------
+
+* **Ray Nodes**: A Ray cluster consists of a **head node** and set of **worker nodes**. The head node needs to be started first, and the worker nodes are given the address of the head node to form the cluster. The Ray cluster itself can also "auto-scale", meaning that it can interact with a Cloud Provider to request or release instances according to application workload.
```

Comment:
```suggestion
* **Ray Nodes**: A Ray cluster consists of a **head node** and a set of **worker nodes**. The head node needs to be started first, and the worker nodes are given the address of the head node to form the cluster. The Ray cluster itself can also "auto-scale," meaning that it can interact with a Cloud Provider to request or release instances according to application workload.
```